<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4,4,4,4,3">
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>144</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>138</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="icon_label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/resource/icons/icon4.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>138</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>109</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="username_lineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>方正姚体</family>
           <pointsize>20</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="username_label">
         <property name="font">
          <font>
           <family>方正姚体</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="password_label">
         <property name="font">
          <font>
           <family>方正姚体</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>密码：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="password_lineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>方正姚体</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,3,1,3,1">
       <item>
        <widget class="QCheckBox" name="type_checkBox">
         <property name="font">
          <font>
           <family>方正姚体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>我是出题人</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="login_pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>方正姚体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>14</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exit_pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>方正姚体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>15</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="4" colspan="2">
        <widget class="QPushButton" name="check_creater_pushButton">
         <property name="font">
          <font>
           <family>方正姚体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>查询出题人</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>方正姚体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>想查询数据？请点击查询</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" colspan="2">
        <widget class="QPushButton" name="registe_user_pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>方正姚体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>注册</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4" colspan="2">
        <widget class="QPushButton" name="regist_creater_pushButton">
         <property name="font">
          <font>
           <family>方正姚体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>注册</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>118</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QPushButton" name="check_user_pushButton">
         <property name="font">
          <font>
           <family>方正姚体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>查询闯关者</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QLabel" name="creater_label">
         <property name="font">
          <font>
           <family>方正姚体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>想成为出题人？没有账号请注册</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QLabel" name="registe_label">
         <property name="font">
          <font>
           <family>方正姚体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>没有游戏账户？请先注册</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>108</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
